src/reactionsystem.jl

```diff
@@ -1500,6 +1526,10 @@ function DiffEqBase.SDEProblem(rs::ReactionSystem, u0, tspan,
                                include_zero_odes = true, checks = false,
                                check_length = false,
                                remove_conserved = false, kwargs...)
+    if !isnothing(noise_scaling) 
+        any(isnoisescalingparameter.(parameters(rs))) && error("You have declared some paraemters as noise scaling parameters, and also given a \"noise_scaling\" argument to SDEProblem. Please remove the \"noise_scaling\", as this way of scaling CLE noise is being depricated.")
```

Comment:
This check should be in `convert` I think, since people may call that manually.

Reply:
In convert it if you add it after flattening on `flatrs` you can just do
```julia
any(isnoisescalingparameter, get_ps(flatrs))
```
to avoid any allocations.

Reply:
I was thinking that this wasn't possible for some reason, but you are totally right, this should be moved to `covert`.